tools/clang/test/CodeGenSPIRV/vertex_shader_derivative_in_branch.hlsl

```diff
@@ -0,0 +1,20 @@
+// RUN: %dxc -T vs_6_0 -E main -fspv-target-env=vulkan1.3 %s -spirv | FileCheck %s
```

Comment:
can you add a negative test?
This way we know the validation error is expected, and can verify we do fail correctly